lib/SPIRV/SPIRVUtil.cpp

```diff
@@ -363,6 +363,11 @@ getSPIRVFuncName(Op OC, StringRef PostFix) {
 }
 
 std::string
+getSPIRVFuncName(Op OC, const Type *pRetTy, bool IsSigned) {
+  return prefixSPIRVName(getName(OC) + kSPIRVPostfix::Divider + getPostfixForReturnType(pRetTy, false));

```

Comment:
Better floow LLVM coding standard http://llvm.org/docs/CodingStandards.html
line width should be < 80